applications/FreeSurfaceApplication/tests/generalTests.py

```diff
@@ -19,10 +19,10 @@ def tearDown(self):
         pass
 
     def testSmallExample(self):
-        self.AssertEqual(True, True)
```

Comment:
ok, even though it does not really belong here

Reply:
I just fixed while was testing all the apps in local, if not they didn't pass the tests :S